interfaces/builtin/lxd_support.go

```diff
@@ -87,6 +102,11 @@ func (iface *lxdSupportInterface) AppArmorConnectedPlug(spec *apparmor.Specifica
 			spec.AddSnippet(lxdSupportConnectedPlugAppArmorWithUserNS)
 		}
 	}
+	var enableUnconfinedMode bool
+	_ = plug.Attr("enable-unconfined-mode", &enableUnconfinedMode)
+	if enableUnconfinedMode {
+		spec.SetUnconfinedEnabled()
```

Comment:
wouldn't this return an error if unconfined isn't supported by the host and lxd has enable-unconfined-mode? if so, perhaps we should at least log something that unconfined mode is ignored

Reply:
No, this call will never fail since the interface sets `appArmorUnconfinedPlugs: true` which is the only prerequisite for this call. Whether the host supports the feature or not is then checked later when actually generating the profile from the spec.

Reply:
I added a comment here in 2f3d61b2c7 to help clarify this. Please let me know if it is still unclear though.

Reply:
This is fine, personally I would use syntax like this, to make it explicit I understand and choose to ignore the error. Some linters understand that as well.

```
_ = spec.SetUnconfiendEnabled()
```